src/schema/term.rs

```diff
@@ -293,7 +304,7 @@ where B: AsRef<[u8]>
     /// Returns `None` if the field is not of string type
     /// or if the bytes are not valid utf-8.
     pub fn as_str(&self) -> Option<&str> {
-        if self.as_slice().len() < 5 {
+        if self.as_slice().len() < TERM_METADATA_LENGTH {
```

Comment:
Is it still possible? After you fix truncate?
If the API enforces this invariant, maybe we can make it an assert instead.

Reply:
yes, I think we can remove it